src/requirements.js

```diff
@@ -0,0 +1 @@
+import * as drag from './gui/dragAndDrop.js';

```

Comment:
don't import gui requirements in requirements.js. This should be a node.js style file and should expose the backend to the gui. Include the gui file in the index like normal